app/src/main/res/layout/screen_patient_edit.xml

```diff
@@ -0,0 +1,229 @@
+<?xml version="1.0" encoding="utf-8"?>
+<org.simple.clinic.editpatient.PatientEditScreen android:id="@+id/patientedit_root"
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent"
+  android:orientation="horizontal">
+
```

Comment:
Are we going to reuse the layout for creating and editing a patient? How do you suggest we reuse things? Do we use styles? Or do we use a single layout?

Reply:
Sure, sounds good